src/tests/work-item/work-item-list/work-item-detailed-dialog.spec.js

```diff
@@ -27,10 +27,61 @@ describe('Work item list', function () {
   var page, items, browserMode;
 
   beforeEach(function () {
-    testSupport.setBrowserMode('phone');
+    testSupport.setBrowserMode('desktop');
     page = new WorkItemListPage(true);   
   });
 
+ it('Create WorkItem and creatorname and image is relecting', function () {
+   page.clickDetailedDialogButton();
+   var detailPage = page.clickDetailedIcon("userstory");
+   detailPage.setWorkItemDetailTitle (workItemTitle, false);
+   detailPage.clickWorkItemTitleSaveIcon();
+   detailPage.clickWorkItemDetailDescription()
+   detailPage.setWorkItemDetailDescription (workItemDescription, true);
+   detailPage.clickWorkItemDescriptionSaveIcon();
+   expect(detailPage.getCreatorUsername()).toBe('Example User 0');
+   expect(detailPage.getCreatorAvatar().isPresent()).toBe(true);     
+   detailPage.clickWorkItemDetailCloseButton();
+   browser.wait(until.presenceOf(page.workItemByTitle(workItemTitle)), waitTime, 'Failed to find workItemList');
+   expect(page.workItemTitle(page.workItemByTitle(workItemTitle))).toBe(workItemTitle);
+ });
+
+ it('Edit and check WorkItem , creatorname and image is relecting', function () {
+   page.clickDetailedDialogButton();
+   var detailPage = page.clickDetailedIcon("userstory");
+   detailPage.setWorkItemDetailTitle (workItemTitle, false);
+   detailPage.clickWorkItemTitleSaveIcon();
+   detailPage.clickWorkItemDetailDescription()
+   detailPage.setWorkItemDetailDescription (workItemDescription, true);
+   detailPage.clickWorkItemDescriptionSaveIcon();
+   expect(detailPage.getCreatorUsername()).toBe('Example User 0');
+   expect(detailPage.getCreatorAvatar().isPresent()).toBe(true);     
+   detailPage.clickWorkItemDetailCloseButton();
+   browser.wait(until.presenceOf(page.workItemByTitle(workItemTitle)), waitTime, 'Failed to find workItemList');
+   expect(page.workItemTitle(page.workItemByTitle(workItemTitle))).toBe(workItemTitle);
+   page.workItemViewId(page.firstWorkItem).getText().then(function (text) { 
+      page.clickWorkItemTitle(page.firstWorkItem, text);
+      expect(detailPage.getCreatorUsername()).toBe('Example User 0');
+      expect(detailPage.getCreatorAvatar().isPresent()).toBe(true);  
+      expect(detailPage.getImageURL()).toBe('https://avatars.githubusercontent.com/u/2410471?v=3&s=20');
+   });
+ });
+ it('check Creator is readonly - desktop', function () {
+   page.clickDetailedDialogButton();
+   var detailPage = page.clickDetailedIcon("userstory");
+   detailPage.clickCreatorDefaultIcon();
+   expect(detailPage.getCreatorDefaultIcon().isPresent()).toBe(true);
+   expect(detailPage.getCreatorUsername()).toBe('No creator found');
```

Comment:
It looks like the error generated by the UI changed - the test will have to also change to stayin synch:

 ```
Work item list check Creator is readonly - desktop
  Message:
    Expected 'Creator not found' to be 'No creator found'.
```